pkg/controller/migmigration/backup.go

```diff
@@ -199,7 +269,12 @@ func (t *Task) getReplicatedBackup() (*velero.Backup, error) {
 	if err != nil {
 		return nil, err
 	}
-	if len(list.Items) > 0 {
+	// If this is stage we only need to wait for 1 backup to be replicated
+	if len(list.Items) == 1 && t.stage() {
```

Comment:
If I'm understanding this correctly, `t.stage()` being set to true will avoid running the `initialBackup`, which is why we only need 1 Backup associated with our Migration (`copyBackup`) to proceed.